index.bs

```diff
@@ -841,7 +861,7 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 							Issue: It should be possible to read the data asynchronously from the system clipboard after the author calls getType, however, this set of steps implies that data will be provided at the time of read.
 						
-						1. The user agent, MAY sanitize |representation|'s [=representation/data=], unless |representation|'s [=representation/MIME type=]'s essence is "image/png", which should remain unsanitized to preserve meta data.
+						1. The user agent, MAY sanitize |representation|'s [=representation/data=], unless |representation|'s [=representation/MIME type=]'s [=MIME type/essence=] is "image/png", which should remain unsanitized to preserve meta data, or equal to |format| if |format| is not empty.
```

Comment:
I don't follow what is meant by ", or equal to |format| if |format| is not empty" and how it relates to the rest of the paragraph. Could you elaborate?

Reply:
It basically means, the user agent may not sanitize the `|representation|'s [=representation/data=]` if `|representation|'s [=representation/MIME type=]'s [=MIME type/essence=]` is equal to `|format|.`

Reply:
Hmm not sure I follow. I understood that the UA must not sanitize for "image/png". Could you give me an example of what you're going for with the second clause?